.github/workflows/main.yml

```
@@ -52,7 +52,7 @@ jobs:
       - name: Run tests
         # Run the tests directly from the source dir so support files (eg, .wav files etc)
         # can be found - they aren't installed into the Python tree.
-        run: python pywin32_testall.py -v -skip-adodbapi
+        run: python -m win32.scripts.pywin32_testall -v -skip-adodbapi
```

```suggestion
        run: python win32/scripts/pywin32_testall.py -v -skip-adodbapi
```
The failing test https://github.com/mhammond/pywin32/actions/runs/12308761196/job/34354669586?pr=2408#step:7:316 and comment 
> \# Run the tests directly from the source dir so support files (eg, .wav files etc)
> \# can be found - they aren't installed into the Python tree.

tell me that it maybe didn't even make sense that the test was shipped in the first place (or could be modified to skip some tests when not run from source)